it/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -0,0 +1,157 @@
+package org.corfudb.universe.scenario.fixture;
+
+import com.google.common.collect.ImmutableList;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import org.corfudb.universe.group.cluster.CorfuClusterParams;
+import org.corfudb.universe.node.client.ClientParams;
+import org.corfudb.universe.node.server.CorfuServer;
+import org.corfudb.universe.node.server.CorfuServerParams;
+import org.corfudb.universe.node.server.vm.VmCorfuServerParams;
+import org.corfudb.universe.universe.UniverseParams;
+import org.corfudb.universe.universe.vm.VmUniverseParams;
+import org.slf4j.event.Level;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Fixture factory provides predefined fixtures
+ */
+public interface Fixtures {
+
+    /**
+     * Common constants used for test
+     */
+    class TestFixtureConst {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)